third_party/openthread/src/ncp/spinel.c

```diff
@@ -1199,6 +1199,22 @@ spinel_prop_key_to_cstr(spinel_prop_key_t prop_key)
         ret = "PROP_THREAD_CONTEXT_REUSE_DELAY";
         break;
 
+    case SPINEL_PROP_THREAD_DISCOVERY_SCAN_JOINER_FLAG:
+        ret = "SPINEL_PROP_THREAD_DISCOVERY_SCAN_JOINER_FLAG";
```

Comment:
These should return just the official name of the property from the Spinel spec, which doesn't include the `SPINEL_` prefix. The `SPINEL_` prefix is just for constants in `spinel.h`.

I won't hold back approval on this though.